addons/l10n_ie/data/template/account.tax-ie.csv

```diff
@@ -1,63 +1,197 @@
+"","","","","","","","","","tax","invoice","","","+T1"
+"","","","","","","","","","base","refund","","",""
+"","","","","","","","","","tax","refund","","","-T1"
+"ie_tax_sale_good_exempt","0% G EXEMPT","0% Goods Exempt","0%","sale","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+"","","","","","","","","","tax","invoice","","","+T1"
+"","","","","","","","","","base","refund","","",""
+"","","","","","","","","","tax","refund","","","-T1"
+"ie_tax_sale_service_exempt","0% S EXEMPT","0% Services Exempt","0%","sale","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+"","","","","","","","","","tax","invoice","","","+T1"
+"","","","","","","","","","base","refund","","",""
+"","","","","","","","","","tax","refund","","","-T1"
+"ie_tax_sale_good_eu_0","0% EU G","0% EU Goods","0%","sale","percent","0","ie_tax_group_vat_0","True","base","invoice","","","+E1"
+"","","","","","","","","","tax","invoice","","","+T1"
+"","","","","","","","","","base","refund","","","-E1"
+"","","","","","","","","","tax","refund","","","-T1"
+"ie_tax_sale_service_eu_0","0% EU S","0% EU Services","0%","sale","percent","0","ie_tax_group_vat_0","True","base","invoice","","","+ES1"
+"","","","","","","","","","tax","invoice","","","+T1"
+"","","","","","","","","","base","refund","","","-ES1"
+"","","","","","","","","","tax","refund","","","-T1"
+"ie_tax_sale_ex_good_0","0% EX G","0% Export Goods","0%","sale","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+"","","","","","","","","","tax","invoice","","","+T1"
+"","","","","","","","","","base","refund","","",""
+"","","","","","","","","","tax","refund","","","-T1"
+"ie_tax_sale_ex_service_0","0% EX S","0% Export Services","0%","sale","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+"","","","","","","","","","tax","invoice","","","+T1"
+"","","","","","","","","","base","refund","","",""
+"","","","","","","","","","tax","refund","","","-T1"
+"ie_tax_purchase_good_23","23% G","23% Standard rate - Goods","23%","purchase","percent","23","ie_tax_group_vat_23","True","base","invoice","","",""
+,"","","","","","","","","tax","invoice","","l10n_ie_account_2131","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","l10n_ie_account_2131","-T2"
+"ie_tax_purchase_service_23","23% S","23% Standard rate - Services","23%","purchase","percent","23","ie_tax_group_vat_23","True","base","invoice","","",""
+,"","","","","","","","","tax","invoice","","l10n_ie_account_2131","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","l10n_ie_account_2131","-T2"
+"ie_tax_purchase_good_eu_23","23% EU G","23% Standard rate - EU - Goods","23%","purchase","percent","23","ie_tax_group_vat_23","True","base","invoice","","","+E2"
+,"","","","","","","","","tax","invoice","","l10n_ie_account_3804","+T1"
```

Comment:
We need the reverse-charge tax liability to credit the VAT payable account (thereby increasing the liability). Same for the corresponding refund repartition line, the other EU purchase taxes below, and the Postponed Accounting import VAT tax.
```suggestion
,"","","","","","","","","tax","invoice","-100","l10n_ie_account_3804","+T1"
```